packages/ContentsColorPickerInput/package.json

```diff
@@ -10,11 +10,11 @@
         "test": "jest",
         "test:watch": "jest --watchAll",
         "coverage": "jest --coverage",
-        "serve": "cross-env WEBPACK_DEV_SERVER=true webpack-dev-server",
-        "serve:https": "cross-env WEBPACK_DEV_SERVER=true RCMS_JS_HTTPS=true webpack-dev-server",
-        "dev": "webpack",
-        "watch": "webpack --watch",
-        "build": "cross-env WEBPACK_MODE=production webpack"
+        "serve": "cross-env WEBPACK_DEV_SERVER=true NODE_OPTIONS=--openssl-legacy-provider webpack-dev-server",
+        "serve:https": "cross-env WEBPACK_DEV_SERVER=true NODE_OPTIONS=--openssl-legacy-provider RCMS_JS_HTTPS=true webpack-dev-server",
+        "dev": "cross-env NODE_OPTIONS=--openssl-legacy-provider webpack",
+        "watch": "cross-env NODE_OPTIONS=--openssl-legacy-provider webpack --watch",
+        "build": "cross-env WEBPACK_MODE=production NODE_OPTIONS=--openssl-legacy-provider webpack"
```

Comment:
NOTE:
This option is for avoiding build failure happened by crypto error, which is a breaking change from Node v17.x.
ref: https://zenn.dev/yogarasu/articles/425732ff408d06